llvm/unittests/Analysis/InlineOrderPlugin/InlineOrderPlugin.cpp

```diff
@@ -0,0 +1,156 @@
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Pass.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Passes/PassPlugin.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include "llvm/Analysis/InlineOrder.h"
+
+using namespace llvm;
+
+namespace {
+
+// This is the implementation used in llvm/lib/Analysis/InlineOrder.cpp
+// Size priority is the default priority used in the inliner.
+class SizePriority {
```

Comment:
I don't think it's a good idea to replicate this here, if there are any changed in the original files then they would break this test. 

Can't you use `createInlineOrder`?  to create this?